card/lib/card/cache.rb

```diff
@@ -18,10 +18,10 @@ class Cache
     TEST_ENVS = %w{test cucumber}
     @@prepopulating     = TEST_ENVS.include? Rails.env
     @@using_rails_cache = TEST_ENVS.include? Rails.env
-    @@prefix_root       = Cardio.config.database_configuration[Rails.env]['database']
+    @@prefix_root       = nil
```

Comment:
Now that I think about it, it does make sense.  All the assignment operators will create the lvalue (a data container) first.  Then it doesn't throw the error because even though it is unassigned, it exists.